BootloaderCommonPkg/Include/Library/PartitionLib.h

```diff
@@ -102,4 +103,19 @@ ClosePartitions (
   IN   EFI_HANDLE             PartHandle
   );
 
+/**
+  Get the GPT entries array from the last GPT block
+  device read.
+
+  @param  None
+
+  @retval                   Pointer to the GPT entry data
+
+**/
+EFI_PARTITION_ENTRY *
+EFIAPI
+GetGptEntryData (
```

Comment:
Considering PartitionLib.h is a library class.    All interfaces for this lib should be partition type independent.  
So we should not provide GPT specific API interface in the library class header.   Either rename this API to be more generic or move it into private header file.